base/strings/util.jl

```diff
@@ -159,28 +176,44 @@ julia> rstrip(a)
 "March"
 ```
 """
-function rstrip(s::AbstractString, chars::Chars=_default_delims)
+function rstrip(f, s::AbstractString)
     for (i, c) in Iterators.reverse(pairs(s))
-        c in chars || return SubString(s, 1, i)
+        f(c) || return SubString(s, 1, i)
     end
     SubString(s, 1, 0)
 end
+rstrip(s::AbstractString) = rstrip(isspace, f)
```

Comment:
A typo? 
`rstrip(s::AbstractString) = rstrip(isspace, f)` should be `rstrip(s::AbstractString) = rstrip(isspace, s)`?

Reply:
Seems `lstrip`  also has this typo.